### PR TITLE
Bugfix/requirements

### PR DESCRIPTION
**Pull request by ahartikainen**
Adding some noise to the commit history.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ numpydoc
 pydocstyle
 pylint
 pyro-ppl
-tensorflow
+tensorflow==1.31.1
```
Comment (ColCarroll) [inline requirements-dev.txt]:
```suggestion
tensorflow==1.13.1
```

Comment (ahartikainen) [reply requirements-dev.txt]: good catch

https://pypi.org/project/tensorflow/#history

It's an image there, so I couldn't copy-paste

**Comment by ahartikainen** [comment]
haha, sorry, I force pushed :)

**Comment by ColCarroll** [comment]
...does this mean i don't get any credit for my work on this PR? :joy_cat:

**Comment by ahartikainen** [comment]
Here you have: 10000 virtual credits

**Pull request merged**